app/views/users/_form_fields.html.erb

```diff
@@ -41,6 +41,12 @@
   <%= f.select :organisation_id, organisation_options(f), organisation_select_options, { class: "chosen-select form-control", 'data-module' => 'chosen' } %>
 </p>
 
+<% if policy(User).flag_2sv? %>
+  <p class="form-group">
+    <%= f.check_box :requires_2sv %> <%= f.label :requires_2sv, "Requires 2SV" %>
```

Comment:
I'll correct this and post a screenshot. This feature is currently available to super-admins when they create an invitation or edit an existing user.

Are there any feels regarding the attribute naming? Is `requires_2v` and `flag_2v?` OK or would you prefer `requires_two_step_verification` and `flag_two_step_verification?`?